### ISSUE TITLE
Missing landmarks and thus possibility to skip document parts

### ISSUE THREAD

**Issue by krallowski** [humhub/humhub#8265]
### What steps will reproduce the problem?
1. Open any page (dashboard/spaces etc)
2. Open the web-developer tools / tab accessibility
3. Check the semantic description of the different marts (navigation / main etc)
4. Search for a "skip to main content" visually hidden

### What is the expected result?
I can skip to main content and the structure of the page (navigation, main content, sidebar etc) is reflected in the html structure either with the html5 elements or aria landmarks (according to: https://bitvtest.de/pruefschritt/bitv-20-plus-web/bitv-20-plus-web-9-2-4-1-bereiche-ueberspringbar)

### What do you get instead?
No semantic structure

### Additional info

I saw a "skip to content" link in the humhub community and a main tag but I did not find it in other humhub instances neither in the code. But maybe I missed it. But in this line it seams that the main html5 tag is missing. But as I said, maybe I missed it?
https://github.com/humhub/humhub/blob/50d0c4b59acbb4f088cea594d1acef098aafb8ab/protected/humhub/views/layouts/main.php#L30

| Q                | A
| ---------------- | ---
| HumHub version   | 1.18
| PHP version      | 
| Operating system |
